index.html

```diff
@@ -0,0 +1,335 @@
+        <span class="badge badge-primary">Primary</span>
+        <span class="badge badge-success">Success</span>
+        <span class="badge badge-warning">Warning</span>
+        <span class="badge badge-danger">Danger</span>
+    </div>
+    <div style="display: flex; align-items: center; justify-content: center; margin: 1rem;">
+        <span class="badge-pills badge-primary">Primary</span>
+        <span class="badge-pills badge-success">Success</span>
+        <span class="badge-pills badge-warning">Warning</span>
+        <span class="badge-pills badge-danger">Danger</span>
+    </div>
+    <div class="badge__on__avatar">
+        <img src="https://pbs.twimg.com/profile_images/1236563952605130752/d9EYzZpu.jpg" alt="" class="avatar">
+        <div class="badge-success badge-round"></div>
+    </div>
+   <div class="badge__on__avatar">
+       <img src="https://pbs.twimg.com/profile_images/1236563952605130752/d9EYzZpu.jpg" alt="" class="avatar avatar-2x">
+       <div class="badge-success badge-round"></div>
+   </div>
+   <div class="badge__on__avatar">
+    <img src="https://pbs.twimg.com/profile_images/1236563952605130752/d9EYzZpu.jpg" alt="" class="avatar avatar-3x">
+    <div class="badge-success badge-round badge-round-2x"></div>
+</div>
+<div class="badge__on__avatar">
+    <img src="https://pbs.twimg.com/profile_images/1236563952605130752/d9EYzZpu.jpg" alt="" class="avatar">
+    <div class="badge-danger badge-round"></div>
+</div>
+<div class="badge__on__avatar">
+   <img src="https://pbs.twimg.com/profile_images/1236563952605130752/d9EYzZpu.jpg" alt="" class="avatar avatar-2x">
+   <div class="badge-danger badge-round"></div>
+</div>
+<div class="badge__on__avatar">
+<img src="https://pbs.twimg.com/profile_images/1236563952605130752/d9EYzZpu.jpg" alt="" class="avatar avatar-3x">
+<div class="badge-danger badge-round badge-round-2x"></div>
+</div>
+<div class="badge-on-icon">
+    <i class="fas fa-home fa-2x text-primary icon-only"></i>
+    <div class="badge-danger badge-pills badge-notification">9+</div>
+</div>
+    <div class="divider"></div>
+
+<!-- BUTTONS -->
+
+    <h3 style="text-align: center; margin: 1rem;">BUTTONS</h3>
+    <div style="margin: 1rem; text-align: center;">
+    <button class="btn btn-primary">
+        Primary
+    </button>
+    <button class="btn btn-primary-success">
+        Success
+    </button>
+    <button class="btn btn-primary-warning">
+        Warning
+    </button>
+    <button class="btn btn-primary-danger">
+        Danger
+    </button>
+    <button class="btn btn-secondary">
+        Primary
+    </button>
+    <button class="btn btn-secondary-success">
+        Success
+    </button>
+    <button class="btn btn-secondary-warning">
+        Warning
+    </button>
+    <button class="btn btn-secondary-danger">
+        Danger
+    </button>
+</div>
+<div class="divider"></div>
+
+
+<!-- LINKS -->
+<h3 style="text-align: center; margin: 1rem;">LINKS</h3>
```

Comment:
**everything looks great to me** 
I think, it's not recommended to use style tags inside HTML! Tags

Reply:
This not for component I just used it to align component name in center